internal/local/dependencies.go

```diff
@@ -38,3 +39,28 @@ func FindAppDependencies(path string) (Dependencies, error) {
 
 	return Dependencies{rootDir, archivePath}, nil
 }
+
+// FindPackageJSON finds the Realm app dependencies as a package.json file
+func FindPackageJSON(path string) (Dependencies, error) {
+	app, appOK, appErr := FindApp(path)
+	if appErr != nil {
+		return Dependencies{}, appErr
+	}
+	if !appOK {
+		return Dependencies{}, nil
+	}
+
+	rootDir := filepath.Join(app.RootDir, NameFunctions)
+
+	packageJSONPath := filepath.Join(rootDir, NamePackageJSON)
+
+	_, err := os.Stat(packageJSONPath)
+	if err != nil {
```

Comment:
[optional nit] I always prefer to keep as tight a scope as possible (especially with the `err` var name)
```suggestion
	if _, err := os.Stat(packageJSONPath); err != nil {
```